framework/object_factory.h

```diff
@@ -14,45 +14,59 @@
  * \param object_name Name of the object in the registry.
  * Example:
  * \code
- * OpenSnRegisterObject(kaka, Zorba);
+ * OpenSnRegisterObjectNamespace(kaka, Zorba);
  * \endcode
  * \note Remember to include the header "framework/object_factory.h".*/
-#define OpenSnRegisterObject(namespace_name, object_name)                                          \
+#define OpenSnRegisterObjectNamespace(namespace_name, object_name)                                 \
   static char OpenSnObjectFactoryJoinWordsB(unique_var_name_object_##object_name##_,               \
                                             __COUNTER__) =                                         \
     opensn::ObjectFactory::AddObjectToRegistry<object_name, Object>(#namespace_name, #object_name)
 
+#define OpenSnRegisterObject(object_name)                                                          \
+  static char OpenSnObjectFactoryJoinWordsB(unique_var_name_object_##object_name##_,               \
+                                            __COUNTER__) =                                         \
+    opensn::ObjectFactory::AddObjectToRegistry<object_name, Object>(#object_name)
+
 /**Macro for registering an object (parameters only) within the
  * ObjectFactory singleton.
  * \param namespace_name Name of the namespace within which the object is.
  * \param object_name Name of the object in the registry.
  * Example:
  * \code
- * OpenSnRegisterObjectParametersOnly(kaka, Zorba);
+ * OpenSnRegisterObjectParametersOnlyNamespace(kaka, Zorba);
```

Comment:
mmmm. can we change the dummy example names used?

Reply:
We can, but they are used in more than just this place and we may be modifying the `ObjectFactory` class in near future, so I would update the doco at that time.

Reply:
Same comment as above. This sounds like we are registering a namespace, not parameters in a namespace.